tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -3,6 +3,8 @@
 
 import cv2
 import numpy as np
+import scipy.ndimage as ndi
```

Comment:
Please check this if it needs to be removed.

Reply:
removed